emissionsapi/utils.py

```diff
@@ -4,9 +4,46 @@
 # See LICENSE fore more information.
 
 
+# user specific exception, used in polygon_to_wkt
+class RESTParamError(ValueError):
+    pass
+
+
 def bounding_box_to_wkt(lon1, lat1, lon2, lat2):
     """Convert a bounding box specified by its top-left and bottom-right
     coordinates to a wkt string defining a polygon.
     """
     return f'POLYGON(({lon1} {lat1},{lon1} {lat2},{lon2} {lat2},'\
            f'{lon2} {lat1},{lon1} {lat1}))'
+
+
+def polygon_to_wkt(polygon):
+    """
+    Takes a point list and converts them to a wkt-polygon.
+    The list of poinst has to be provided in the form of
+    (lon1, lat1, lon2, lat2, ...).
+    Therefor, the number of elements in the list has to be even.
+    If the polygon is not closed (the first lon/lat pair does not match
+    the last lon/lat pair), the polygon will be closed here.
+    """
```

Comment:
We are using the sphinx docstrings for documentation, see https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html

What do you think about

```
"""Converts a list of points to a WKT string defining a polygon.

:param polygon: List of values and every pair of values represent a consecutive vertex of the polygon.
:type polygon: list
:return: WKT String defining the polygon.
:rtype: str
"""
```

Reply:
Thanks for the input, I like that, didnt know the syntax 👍